contents/blog/customer-churn-analysis-guide.md

```diff
@@ -0,0 +1,253 @@
+
+### The things you need to improve retention
+
+You're gonna need a few things to track, analyze and reduce customer churn, such as:
+
+1. The ability to identify your new/current users (e.g. job role, location etc.)
+
+1. An analytics tool to help you track user churn, and analyze use behavior.
+
+1. Session recording, so you can watch real interactions with your product.
+
+1. A way to test new features and improvements to validate if they work.
+
+1. The ability to safely roll out new features to test groups and specific cohorts
+
+1. Product-minded engineers or product managers who talk to users and ship fast
+
+Some companies deploy a complicated data stack to get all this, but it's not necessary. We built PostHog (yes, this is a teeny bit of sales pitch, now) so you can get all those things in one, tightly integrated tool.
+
+And, while we can't help you with #6, we do have strong opinions on why [product engineers are awesome](/blog/what-is-a-product-engineer) for startups, and [how product engineers and product managers](/blog/product-engineer-vs-product-manager) can work together.
+
+Check out [our product page](/product) if this sounds interesting to you.
+
+/pitch over
+
+## Takeaways
+
+Phew, ok, that was lot. Hopefully you know a little more about how churn and retention rate work, interact, and impact your product now.
+
+To conclude, here are {CHANGE THIS} takeaways you can use to sound smart in meetings, or perhaps share on LinkedIn 😱 with a hashtag – I won't judge, not publicly at least:
+
+1. Early-stage startups shouldn't get too hung up on how high or low churn is, but you should care **which direction** it's headed. The actual rate of churn / retention is more important once you have product-market fit and care about revenue.
+
+1. Don't benchmark your churn rate against the wrong industry, or product type. Average revenue per customer (ARPC) is a huge factor here. A B2B SaaS with an ARPC of $200 per month should have very different expectations than one with an ARPC of $1,000 per month.
+
+1. PostHog is awesome and you should totally adopt it at your startup. "Nobody got fired for buying PostHog" said someone, probably me.
+
+- TODO: WILL PROBABLY ADD ANOTHER TWO HERE
+
+### Further reading
+
+If you're interested in early-stage startups and product-led growth, you may find these articles useful: 
+
+- PostHog CEO, James Hawkins, on [how we found our Ideal Customer Profile](/blog/creating-ideal-customer-profile)
+
+- My guide to [building an AARRR pirate funnel](/blog/aarrr-pirate-funnel) for tracking your most important metrics
+
+- Another piece from James on [how we made something people want](/blog/making-something-people-want)
+
+You can also sign up to our newsletter over there 👉. We send it every two weeks. It's full of articles like this.
```

Comment:
```suggestion
You can also sign up to our newsletter over there 👉 (or 👇 on mobile). We send it every two weeks. It's full of articles like the one you just read.
```

Reply:
Hmm, good spot. Might be easier to turn this into a link.